charts/trino/values.yaml

```diff
@@ -830,6 +895,8 @@ serviceMonitor:
     prometheus: kube-prometheus
   # serviceMonitor.interval -- The serviceMonitor web endpoint interval
   interval: "30s"
+  # serviceMonitor.path -- HTTP path from which to scrape for metrics.
+  path: /metrics
```

Comment:
https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#endpoint says this is the default. What's the use case for making it configurable?

Reply:
Now that you point it out I guess it was just an excess of zeal on my part :)